.claude/skills/contributing/references/testing.md

```
@@ -0,0 +1,89 @@
+# Test & Verify TEDI Component
+
+Target: `$ARGUMENTS`
+
+## Workflow
+
+### 1. Run Tests
+
+If a path was provided, run tests for that specific component:
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)